snapcraft/application.py

```diff
@@ -224,21 +224,21 @@ def create_app() -> Snapcraft:
     app.add_command_group(
         "Store Snap Release Management",
         [
-            unimplemented.Release,
-            unimplemented.Close,
-            unimplemented.Status,
-            unimplemented.Upload,
+            commands.StoreReleaseCommand,
+            commands.StoreCloseCommand,
+            commands.StoreStatusCommand,
+            commands.StoreUploadCommand,
             unimplemented.Push,
             unimplemented.Promote,
```

Comment:
why not promote?

Reply:
There are 11 legacy store commands that I was going to do in a separate PR.  Looking at them now, it looks straightforward enough. I'll try to fit it into this PR.

Reply:
Added!